.github/workflows/runtests.yml

```diff
@@ -184,7 +184,11 @@ jobs:
                 sudo -H pip3 install --upgrade --ignore-installed pyyaml
          - name: "Install latest doxygen . . ."
            run: |
-                bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+                sudo rm -rf /usr/local/include/node
+                sudo rm -f /opt/pipx_bin/ansible-config
+                /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
```suggestion
```
This shouldn't be necessary since `brew` is pre-installed on GitHub CI.

Try running this job on `ubuntu-22.04`.